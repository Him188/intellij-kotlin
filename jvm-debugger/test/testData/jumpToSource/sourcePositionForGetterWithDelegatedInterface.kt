package sourcePositionForGetterWithDelegatedInterface

interface Interface {
    val iInt get() = 10
}

class Delegate : Interface {
}

class TestClass(delegate: Delegate) : Interface by delegate {
}

fun main() {
    val instance = TestClass(Delegate())
    // JUMP TO SOURCE: instance.iInt
    //Breakpoint!
    println("")
}