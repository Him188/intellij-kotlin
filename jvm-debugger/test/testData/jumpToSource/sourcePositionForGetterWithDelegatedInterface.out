LineBreakpoint created at sourcePositionForGetterWithDelegatedInterface.kt:17
Run Java
Connected to the target VM
sourcePositionForGetterWithDelegatedInterface.kt:17
Source position of instance.iInt is 4
Disconnected from the target VM

Process finished with exit code 0

